AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.kt

```diff
@@ -461,6 +464,7 @@ open class DeckPicker :
         Onboarding.DeckPicker(this, mRecyclerViewLayoutManager).onCreate()
 
         launchShowingHidingEssentialFileMigrationProgressDialog()
+        checkWebviewVersion()
```

Comment:
Just a thought on performance - this is intended to be a developer warning only, and I wonder if it makes sense to only execute it on debug builds to make sure there is zero impact on release builds / regular users? Something like this?

```suggestion
        if (BuildConfig.DEBUG) {
            checkWebviewVersion()
        }
```

What do you think?

Reply:
Yes that makes sense! Thanks